tests/subsys/zigbee/osif/nvram/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+endif()
```

Comment:
Remove those lines, and rename the file to `app.overlay` instead.

Reply:
fixed

Reply:
Lines are still present.